README.md

```diff
@@ -1,24 +1,19 @@
 # Acrolinx Java SDK
 
-[![Maven Central](https://img.shields.io/maven-central/v/com.acrolinx.client/sdk)](https://search.maven.org/artifact/com.acrolinx.client/sdk)
-![Build Status](https://github.com/acrolinx/sdk-java/actions/workflows/build-deploy.yml/badge.svg)
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=acrolinx_sdk-java&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=acrolinx_sdk-java)
+[![Build](https://github.com/acrolinx/sdk-java/actions/workflows/build.yml/badge.svg)](https://github.com/acrolinx/sdk-java/actions/workflows/build.yml)
 
-
-This library is meant to be used to interact with the Acrolinx Platform API in automated integrations.
-It does NOT offer an interface to work with the Acrolinx Sidebar (see [Sidebar Java SDK](https://github.com/acrolinx/sidebar-sdk-java)).
+This is a Java library to interact with the Acrolinx Platform API.
+It does NOT offer graphical user interface (GUI) to work with the Acrolinx Sidebar (see [Sidebar Java SDK](https://github.com/acrolinx/sidebar-sdk-java)).
```

Comment:
Why the uppercase NOT? I don't think that it is very necessary because the name of the library doesn't indicate that there is a GUI. 

Also, I think that you could add the "a" or "the" articles before "graphical user interface (GUI)...".

Reply:
Thanks, fixed!